src/classes/HH_NameSpec.cls

```diff
@@ -217,76 +216,102 @@ public with sharing class HH_NameSpec implements HH_INaming {
     * return string
     */
     private static string strNameFromNameSpec(NameSpec ns, list<Contact> listCon) {
-        string strName = '';
-        integer cCon = listCon.size();          
-        integer i = 0;
-        boolean isNoLastnames = (ns.strFullNameSpec.indexOf(chLToken + 'LastName' + chRToken) == -1);
+        Integer size = listCon.size();    
```

Comment:
I'd prefer variables don't get renamed in existing code.  It makes the diff more complicated, and it changes the original author's intent.  In this case the variable name cCon represents the count of Contacts, which is more descriptive and useful in my opinion, than the generic name size.